util/chplenv/compiler_utils.py

```diff
@@ -14,7 +14,9 @@
 @memoize
 def get_compiler_version(compiler):
     version_string = '0'
-    if 'gnu' in compiler:
+    if compiler == 'aarch64-gnu':
+        version_string = run_command(['aarch64-unknown-linux-gnu-gcc', '-dumpversion'])
```

Comment:
The actual compiler name is aarch64-unknown-linux-gnu-gcc.  There is no gcc-cross-aarch64 on either of the configurations of the port.